### PR TITLE
Reflection

### PR DESCRIPTION
**Pull request by a-likhtar**

### CODE REVIEW

File: 10-Reflection/Reflection.Tasks/CommonTasks.cs [hunk 1]
```diff
@@ -59,9 +83,23 @@ public static class CommonTasks
...
+            if (parts.Length > 1)
+            {
+                path = parts[parts.Length - 1];
+                parts = parts.TakeWhile((p, i) => i < parts.Length - 1).ToArray();
+
+                string path2 = String.Join(".",parts);
+                root = obj.GetPropertyValue<object>(path2);
+            }
+
+            var property = root.GetType().BaseType.GetProperty(path);
```
Comment (AlexMokin) [inline 10-Reflection/Reflection.Tasks/CommonTasks.cs]: https://allisonmaruska.files.wordpress.com/2015/10/we-need-to-go-deeper.jpg

Comment (a-likhtar) [reply 10-Reflection/Reflection.Tasks/CommonTasks.cs]: works fine with more property depth :)
either using recursion here a bad practice?)
just add some test with additional property for Manager class

Comment (AlexMokin) [reply 10-Reflection/Reflection.Tasks/CommonTasks.cs]: recursion is certainly a bad practice and can be easily avoided here, but i meant deeper in a class hierarchy

**Comment by AlexMokin** [comment]
good job!

**Pull request closed without merging**